site/_includes/partials/navigation-rail.njk

```diff
@@ -3,10 +3,10 @@
 <navigation-rail role="navigation" class="lg:pad-left-200 lg:pad-right-200" aria-label="{{ 'i18n.common.primary' | i18n(locale) }}" tabindex="-1">
   <div class="display-flex lg:display-none align-center pad-200 hairline-bottom">
     <button class="navigation-rail__close button display-flex align-center justify-content-center width-700 height-700" aria-label="close">
-      {{ icon('close') }}
+      {{ icon('close', {label: 'close'}) }}
     </button>
     <div class="display-flex pad-left-200 pad-right-200" class="navigation-rail__head">
-      {{ icon('logo') }}
+      {{ icon('logo', {label: 'Google Chrome logo'}) }}
```

Comment:
Should this label be localized?

Reply:
Yes, but, I've removed "logo", so it won't need it (since we don't translate the browser name).